components/renku_data_services/project/api.spec.yaml

```diff
@@ -515,6 +541,16 @@ components:
             description: A flag to filter projects where the user is a direct member.
             type: boolean
             default: false
+    ProjectPermissions:
+      description: The set of permissions on a project
+      type: object
+      properties:
+        write:
+          description: The user can edit the project
+          type: boolean
+        admin:
+          description: The user can delete the project and manage members
+          type: boolean
```

Comment:
suggestion: I think it is better if this follows the scopes we have in authz.models more closely.

And currently the Scope enum is:

```
class Scope(Enum):
    """Types of permissions - i.e. scope."""

    READ: str = "read"
    WRITE: str = "write"
    DELETE: str = "delete"
    CHANGE_MEMBERSHIP: str = "change_membership"
    READ_CHILDREN: str = "read_children"
```

So it is better to replace `admin` with `delete` and `change_membership`.